src/components/AppContent/AppContent.vue

```diff
@@ -21,78 +23,59 @@
  -->
 
 <template>
-	<main id="app-content" class="no-snapper" :style="opened ? 'transform: translateX(300px)' : ''">
-		<AppNavigationToggle :aria-expanded="opened"
-			aria-controls="app-navigation"
-			@click="toggleNavigation" />
+	<main id="app-content" class="no-snapper">
 		<!-- @slot Provide content to the app content -->
 		<slot />
 	</main>
 </template>
 
 <script>
 import Hammer from 'hammerjs'
-import AppNavigationToggle from '../AppNavigationToggle'
+import { emit } from '@nextcloud/event-bus'
 
 /**
  * App content container to be used for the main content of your app
  *
  */
 export default {
 	name: 'AppContent',
-	components: {
-		AppNavigationToggle,
-	},
-	data() {
-		return {
-			// closed by default on mobile mode
-			opened: false,
-		}
-	},
+
 	mounted() {
 		this.mc = new Hammer(this.$el, { cssProps: { userSelect: 'text' } })
 		this.mc.on('swipeleft swiperight', e => {
```

Comment:
```suggestion
		this.mc.on('swipeleft swiperight', this.handleSwipe)
```

and `this.mc.off(...) in unmounted

Reply:
Done!